README.md

```diff
@@ -187,3 +187,109 @@ def hello_world():
 if __name__ == '__main__':
     app.run()
 ```
+
+**Integrating with Django**
+
+Place the following in your `settings.py` file:
+
+```python
+# If on Django < 1.10
+MIDDLEWARE_CLASSES = [
+    # ... or whatever is below for you ...
+    'django.middleware.security.SecurityMiddleware',
+    'django.contrib.sessions.middleware.SessionMiddleware',
+    'django.middleware.common.CommonMiddleware',
+    'django.middleware.csrf.CsrfViewMiddleware',
+    'django.contrib.auth.middleware.AuthenticationMiddleware',
+    'django.contrib.auth.middleware.SessionAuthenticationMiddleware',
+    'django.contrib.messages.middleware.MessageMiddleware',
+    'django.middleware.clickjacking.XFrameOptionsMiddleware',
+    # ... or whatever is above for you ...
+    'applicationinsights.django.ApplicationInsightsMiddleware',   # Add this middleware to the end
+]
+
+# If on Django >= 1.10
+MIDDLEWARE = [
+    # ... or whatever is below for you ...
+    'django.middleware.security.SecurityMiddleware',
+    'django.contrib.sessions.middleware.SessionMiddleware',
+    'django.middleware.common.CommonMiddleware',
+    'django.middleware.csrf.CsrfViewMiddleware',
+    'django.contrib.auth.middleware.AuthenticationMiddleware',
+    'django.contrib.messages.middleware.MessageMiddleware',
+    'django.middleware.clickjacking.XFrameOptionsMiddleware',
+    # ... or whatever is above for you ...
+    'applicationinsights.django.ApplicationInsightsMiddleware',   # Add this middleware to the end
+]
+
+APPLICATION_INSIGHTS = {
+    # (required) Your Application Insights instrumentation key
+    'ikey': "00000000-0000-0000-0000-000000000000",
+    
+    # (optional) By default, if DEBUG is True, then the middleware will
+    # not log any events.  To override this behavior, set debug_ikey, below:
+    'debug_ikey': "00000000-0000-0000-0000-000000000000",
+    
+    # (optional) By default, request names are logged as the fully-qualified
+    # name of the view.  To disable this behavior, specify:
+    'use_operation_url': True,
```

Comment:
I think we should be consistent with ASP.NET and Node.JS and log "GET /Index" or Request URL as full URL